src/util/Taggable.h

```diff
@@ -226,15 +226,18 @@ class TagDecoratorFactory final
         auto const& style = value.as_string();
 
         if (boost::iequals(style, "int") || boost::iequals(style, "uint"))
+        {
```

Comment:
curious why this if needs brackets?

Reply:
It actually doesn't need. It was automatically fixing brackets first when there was `else` but when I was removing `else` after `return`, I missed  deleting some brackets. I will fix all the places you found.